.github/workflows/codecov.yaml

```diff
@@ -19,14 +19,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: ["mps2/an385", "native_sim", "qemu_x86", "unit_testing"]
+        platform: ["mps2/an385", "native_sim/native", "qemu_x86/atom", "unit_testing"]
```

Comment:
specially as with this change twister would not anymore allow set as target the boards without the SoC name.